index.html

```diff
@@ -0,0 +1,545 @@
+                        Textbox
+                    </li>
+                    <li>
+                        Validation and Errors styled
+                    </li>
+                </ul>
+                <a href="#lists-id">
+                    <li> Lists</li>
+                </a>
+                <ul>
+                    <li>
+                        Spaced
+                    </li>
+                    <li>
+                        Stacked
+                    </li>
+                </ul>
+                <a href="#navigation-id">
+                    <li> Navigation</li>
+                </a>
+                <a href="#modal-id">
+                    <li> Modal</li>
+                </a>
+                <a href="#rating-id">
+                    <li> Rating </li>
+                </a>
+                <a href="#grid-id">
+                    <li> Simplified Grid</li>
+                </a>
+                <ul>
+                    <li>
+                        grid-2: two items in a grid
+                    </li>
+                    <li>
+                        grid-3: three items
+                    </li>
+                </ul>
+                <a href="#text-id">
+                    <li> Text Utilities</li>
+                </a>
+                <ul>
+                    <li>
+                        Headings
+                    </li>
+                    <li>
+                        Small Text
+                    </li>
+                    <li>
+                        Gray Text
+                    </li>
+                    <li>
+                        Center Text
+                    </li>
+                </ul>
+                <a href="#toast-id">
+                    <li> Toast</li>
+                </a>
+            </ul>
+        </nav>
+        <div class="main">
+            <h1>Installation</h1>
+            <p>Copy and paste the code in the head tag of your html document. So that you can style your components by
+                just adding class names to your html elements. To know about the styled components, their variations,
+                read the documentation
+            </p>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%2823%2C23%2C23%2C1%29&t=blackboard&wt=none&l=htmlmixed&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fbulsaraui.netlify.app%252Fstyles.css%2522%253E"
+                style="width: 650px; height: 100px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <h1 id="avatar-id">Avatar</h1>
+            <p><img src="./images/img_avatar.png" alt="Avatar" class="avatar">
+                <img src="./images/img_avatar2.png" alt="Avatar" class="avatar">
+            </p>
+            <br>
```

Comment:
This may not be a good practice to use the ```br``` in such a way.

Ref this: https://stackoverflow.com/questions/462619/should-br-and-hr-be-avoided-at-all-costs-in-web-design/462703